config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :sightings
+resources :animals
```

Comment:
Check indentation

Reply:
Thanks Elyse. I fixed the indentation. I originally had it on different branches but when I pulled from main, some stuff from Animals like my database and READme did not come through. I had to remake my database again so that's  why it's on the same branch.